src/core/api/models/admin_job.go

```diff
@@ -44,13 +44,18 @@ const (
 
 // AdminJobReq holds request information for admin job
 type AdminJobReq struct {
-	Name       string                 `json:"name"`
-	Schedule   *ScheduleParam         `json:"schedule"`
+	Name string `json:"name"`
+	AdminJobSchedule
```

Comment:
If `AdminJobReq` is to shadow the struct `AdminJobSchedule`, could you put `AdminJobSchedule` ahead of the other typed attributes, in this case `Name`?